### PR TITLE
Added "HD" to the HD channels

### PR DESCRIPTION
**Pull request by wuniversales**

### CODE REVIEW

File: channels/es.m3u [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guidespain.xml.gz"
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas HD
```
Comment (freearhey) [inline channels/es.m3u:2]: I don't have this channel in HD quality at all:
Captura de pantalla 2020-04-29 a las 14 56 13

File: channels/es.m3u [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 http://hlsliveamdgl8-lh.akamaihd.net/i/hlsdvrlive_1@583030/index_1500_av-p.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/SvUWRQm.png" group-title="News",24 Horas
 http://hlsliveamdgl8-lh.akamaihd.net/i/hlsdvrlive_1@583030/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/sADbx7S.png" group-title="",25 TV
 http://cdnlive.shooowit.net/25televisiolive/25tv.smil/.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/M0JqHa9.png" group-title="Local",7 Murcia
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://i.imgur.com/M0JqHa9.png" group-title="Local",7 Murcia HD
```
Comment (freearhey) [inline channels/es.m3u:8]: Same here:
Captura de pantalla 2020-04-29 a las 14 58 01

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

**Comment by picoluna1969** [comment]
Cool

**Comment by picoluna1969** [comment]
Vool

**Comment by picoluna1969** [comment]
Super bedankt

File: channels/es.m3u (hunk 2 above)
Comment (wuniversales) [reply channels/es.m3u:8]: They are supposed to be HD channels, if they don't broadcast in HD, what are we going to do ...
Equally, if they are not HD.
Why repeat the channel in the list with the same name?
In such a case you would simply delete the repeated line.

Comment (freearhey) [reply channels/es.m3u:8]: I think you're right, the second line should be deleted.

**Pull request closed without merging**